app/redhat_insights/default/data/ui/views/events.xml

```diff
@@ -14,16 +14,17 @@ index=redhatinsights
+| eval title=case(event_type=="new-advisory",'payload.synopsis', event_type=="new-recommendation",'payload.rule_description', event_type=="resolved-recommendation",'payload.rule_description', event_type=="deactivated-recommendation", 'payload.rule_description', event_type=="policy-triggered", 'payload.policy_name'." policy triggered", event_type=="drift-baseline-detected","Drift detected from ".'payload.baseline_name'." baseline", event_type=="compliance-below-threshold",'payload.policy_name', event_type=="report-upload-failed", 'payload.error', !isnull(event_type), 'event_type') 
 | eval time=strptime(timestamp, "%Y-%m-%dT%H:%M:%S.%3N")
-| eval url = case(event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="report-upload-failed", INVENTORY_URL.'payload.host_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
-| eval event_type = case(event_type=="resolved-recommendation", "Resolved recommendation", event_type=="new-recommendation", "New recommendation", event_type=="deactivated-recommendation", "Deactivated recommendation", event_type=="policy-triggered","Policy triggered", event_type=="drift-baseline-detected","Drift from baseline detected", event_type=="compliance-below-threshold","System is non compliant to SCAP policy", event_type=="report-upload-failed","Policy report failed to upload", !isnull(event_type), 'event_type')
+| eval url = case(event_type=="new-advisory", ADVISORY_URL.'payload.advisory_name', event_type=="any-cve-known-exploit ", CVE_URL.'payload.reported_cve', event_type=="drift-baseline-detected", DRIFT_URL.'payload.baseline_id', event_type=="report-upload-failed", INVENTORY_URL.'payload.host_id', event_type=="policy-triggered", POLICIES_URL.'payload.policy_id', event_type="compliance-below-threshold", COMPLIANCE_URL.'payload.host_id', !isnull('payload.rule_url'), 'payload.rule_url')
+| eval event_type = case(event_type=="any-cve-known-exploit", "CVEs with known exploits", event_type=="new-advisory", "New Advisory", event_type=="resolved-recommendation", "Resolved recommendation", event_type=="new-recommendation", "New recommendation", event_type=="deactivated-recommendation", "Deactivated recommendation", event_type=="policy-triggered","Policy triggered", event_type=="drift-baseline-detected","Drift from baseline detected", event_type=="compliance-below-threshold","System is non compliant to SCAP policy", event_type=="report-upload-failed","Policy report failed to upload", !isnull(event_type), 'event_type')
```

Comment:
There are more Vulnerability event types that need description. Check https://github.com/RedHatInsights/vulnerability-engine/blob/da9d8c00550059338970b1f6c417ed27053a295e/common/peewee_model.py#L17-L23

Reply:
please check the event descriptions with Vulnerability team. For example new-cve-cvss is not just "CVE with CVSS". Every CVE has CVSS.